pkgs/by-name/ta/tailscalesd/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub
+}:
+
+buildGoModule rec {
```

Comment:
buildGoModule doesn't support finalAttrs yet, but a let binding for version might still be cleaner then using rec here.

Reply:
using rec here is fine.